tests/gordo_components/server/test_gordo_server.py

```diff
@@ -109,8 +111,9 @@ def test_list_revisions(tmpdir, revisions: List[str]):
         client = app.test_client()
         resp = client.get("/gordo/v0/test-project/revisions")
 
-    assert set(resp.json.keys()) == {"latest", "available-revisions"}
-    assert resp.json["latest"] == model_dir
+    assert set(resp.json.keys()) == {"latest", "available-revisions", "revision"}
+    assert resp.json["latest"] == os.path.basename(model_dir)
+    assert resp.json["revision"] == os.path.basename(model_dir)
```

Comment:
Is there a scenario where the client could not be *latest*, but it is aware of *latest*? If so, maybe a test for that combo to be sure it's working.

Reply:
Extended the test to do as you suggest. :+1: